cmd/run.go

```diff
@@ -399,7 +428,7 @@ func (r *Runner) restoreJob(jobID string) error {
 Runs a job for the self serve model of Cedana.
 We don't deploy an orchestrator to the cloud (instead we run it locally in a daemon) and pass through our NATS.
 */
-func (r *Runner) runJobSelfServe() error {
+func (r *Runner) runJobSelfServe(isTest ...bool) error {
```

Comment:
Can you make this an envvar instead of an argument passed to the function? Will be cleaner to set CEDANA_INTEGRATION=true somewhere or something and check against that.